AndroidSDKTests/build.gradle

```diff
@@ -50,7 +50,7 @@ dependencies {
 
     // Dependencies used for unit tests.
     testImplementation 'junit:junit:4.12'
-    testImplementation('org.robolectric:robolectric:3.5.1') {
+    testImplementation('org.robolectric:robolectric:4.0-beta-1') {
```

Comment:
We are using a beta version? Why is this necessary? There is no description on this PR, no issue/ticket. We can't just arbitrarily upgrade to beta versions.